src/gantry/widgets.py

```
@@ -461,9 +466,225 @@ def _build_text(self) -> str:
+                table = self.screen.query_one(ResourceTable)
+                table.focus()
+            except NoMatches:
+                pass
+        elif event.key == "enter":
+            event.stop()
+            self.remove_class("show")
+            try:
+                table = self.screen.query_one(ResourceTable)
+                table.focus()
+            except NoMatches:
+                pass
```

Ruff (0.15.12)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)